Sources/ArcGISToolkit/Resources/en.lproj/Localizable.strings

```diff
@@ -128,6 +134,9 @@
 /* No comment provided by engineer. */
 "Mode" = "Mode";
 
+/* Shorthand for Not Available */
+"N/A" = "N/A";
```

Comment:
Should we use `Not Available` here? Not sure if there is an easy way to localize `N/A` to other languages.

Reply:
That seems like a good idea considering it has [various meanings](https://en.wikipedia.org/wiki/N/A):

> N/A or sometimes n/a is a common [abbreviation](https://en.wikipedia.org/wiki/Abbreviation) in [tables](https://en.wikipedia.org/wiki/Table_(information)) and lists for the phrase not applicable,[[1]](https://en.wikipedia.org/wiki/N/a#cite_note-ODA-1)[[2]](https://en.wikipedia.org/wiki/N/a#cite_note-Foley-2)[[3]](https://en.wikipedia.org/wiki/N/a#cite_note-Kay-3)[[4]](https://en.wikipedia.org/wiki/N/a#cite_note-Elia-4) not available,[[3]](https://en.wikipedia.org/wiki/N/a#cite_note-Kay-3)[[4]](https://en.wikipedia.org/wiki/N/a#cite_note-Elia-4) not assessed, [[5]](https://en.wikipedia.org/wiki/N/a#cite_note-5) or no answer.